pkg/private/BUILD

```diff
@@ -55,6 +55,10 @@ exports_files(
 config_setting(
     name = "private_stamp_detect",
     values = {"stamp": "1"},
+    # Depended on by //src/main/java/com/google/devtools/build/lib/skyframe/serialization/autocodec:bootstrap_autocode in @bazelbuild/bazel.
```

Comment:
Ugg.  
I don't think we need to refer to the acutal file, just that --incompatible_config_setting_private_default_visibility means the flag has to be public.

Reply:
How's this?